build.gradle

```diff
@@ -98,14 +99,26 @@ task downloadLiferayDefinitions(type: Download){
 	overwrite false
 }
 
+task downloadLiferayTaglibDefinitions(type: Download){
+	def target = new File("src/main/resources/tld");
```

Comment:
This shouldn't copy into the source directory, since we don't want them checked into source.  instead, they should be copied into an output folder location and then included into the jar.